lang/java/avro/src/main/java/org/apache/avro/Schema.java

```diff
@@ -1450,7 +1473,7 @@ private static Schema applyAliases(Schema s, Map<Schema,Schema> seen,
     case ENUM:
       if (aliases.containsKey(name))
         result = Schema.createEnum(aliases.get(name).full, s.getDoc(), null,
-                                   s.getEnumSymbols());
+                                   s.getEnumSymbols(), s.getEnumDefault());
```

Comment:
I scanned for other places where Schema.createEnum is called to see which need updates.
ResolvingVisitor.java also needs this same change.  This is used to support IDL.  We should also add support to IDL for enum defaults.
SchemaBuilder.java should be changed to permit specifying an enum default.

Reply:
Okay yes, I can add that to this PR.